### ISSUE TITLE
Unable to download JSON metadata: HTTP Error 404: Not Found

### ISSUE THREAD

**Issue by brunoVanDame** [ytdl-org/youtube-dl#30923]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.imdb.com/video/vi2580586009']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.6.9 (CPython) - Linux-4.15.0-176-generic-x86_64-with-LinuxMint-19-tara
[debug] exe versions: ffmpeg 3.4.8, ffprobe 3.4.8
[debug] Proxy map: {}
[imdb] 2580586009: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/extractor/common.py", line 634, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/usr/lib/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.6/urllib/request.py", line 570, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)

## Description

When executing the command line : 

/usr/local/bin/youtube-dl -v https://www.imdb.com/video/vi2580586009

the problem occurs.

**Comment by dirkf** [comment]
Are you using the release version or the patch from your issue #30824?

**Comment by rgoubet** [comment]
Same for me:

```
$ youtube-dl -F --verbose https://www.arte.tv/fr/videos/100225-000-A/sagrada-familia-le-defi-de-gaudi/
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-F', '--verbose', 'https://www.arte.tv/fr/videos/100225-000-A/sagrada-familia-le-defi-de-gaudi/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.9.12 (CPython) - Linux-5.15.0-27-generic-x86_64-with-glibc2.35
[debug] exe versions: ffmpeg 4.3.2, ffprobe 4.3.2
[debug] Proxy map: {}
[ArteTV] 100225-000-A: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/home/raphael/miniconda3/lib/python3.9/site-packages/youtube_dl/extractor/common.py", line 634, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/home/raphael/miniconda3/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/home/raphael/miniconda3/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/home/raphael/miniconda3/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/home/raphael/miniconda3/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/home/raphael/miniconda3/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/home/raphael/miniconda3/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
```

Using the [package from conda-forge](https://anaconda.org/conda-forge/youtube-dl).

**Comment by dirkf** [comment]
>Same for me:
>
>$ youtube-dl -F --verbose https://www.arte.tv/fr/videos/100225-000-A/sagrada-familia-le-defi-de-gaudi/
...

Plainly not relevant since Arte is not IMDB. See #30878.

**Comment by dirkf** [comment]
The IMDB link works with the yt-dlp extractor. The [yt-dl patch](https://github.com/ytdl-org/youtube-dl/issues/30824#issuecomment-1090310225) uses the same approach, so it could work too. For code coherence, the yt-dlp extractor should be back-ported.

**Comment by rgoubet** [comment]
Plainly relevant since it's the same issue on a different site, furthermore indicating that more sites could be affected.

I'm hoping a new release will include this fix soon and be added to conda package.

**Comment by brunoVanDame** [comment]
I have verified both patches once again mentioned in issue #30824 and both suffer the same problem.

**Comment by dirkf** [comment]
>Plainly relevant ...

No. As with families in Anna Karenina, working extractors are all alike, but each failing extractor fails in its own way, even if the reported diagnostic looks similar.

>I have verified both patches once again mentioned in issue #30824 and both suffer the same problem.

Thanks. The back-port will solve the issue, anyway.

**Comment by benjibasson83** [comment]
Hi
this bug still occure althought this ticket has been closed and marked as solved ( without any understandable explain)
where is the back-port will solve the issue?
how to solve the issue?
quotation of command line please
`youtube-dl https://www.arte.tv/fr/videos/101322-000-A/parasite/
[ArteTV] 101322-000-A: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
`

**Comment by dirkf** [comment]
The issue is about IMDB, not Arte.fr. See https://github.com/ytdl-org/youtube-dl/issues/30923 for Arte.

This issue for IMDB remains open since the symptom is not identical to that in #30824, but the same fix will close both issues when implemented

**Comment by ygoe** [comment]
Same issue here: `youtube-dl --verbose https://www.arte.tv/de/videos/108829-001-A/diener-des-volkes-staffel-2-1-23/`

So this issue is about IMDB, not Arte, *and* this issue is about Arte? I'm confused. Posting this info anyway.

Any workaround known?

**Comment by dirkf** [comment]
The Arte issue is #30923, so read that. Any work-around (there isn't one) would be described or linked from there.

**Comment by ygoe** [comment]
I'm not sure what you're talking about. *This is* issue 30923, and you're pointing me *away* to the same issue 30923? Still confused. Please decide what to say.

**Comment by dirkf** [comment]
Sorry, obvs meant #30919, thanks for the hint.

**Comment by dirkf** [comment]
See https://github.com/ytdl-org/youtube-dl/issues/30878#issuecomment-1141949627.

**Comment by brunoVanDame** [comment]
Jep, am using.
The error is different from the one mentioned in the issue.
Checked that already.
________________________________
